Vizzuality/heco-invest
# API endpoint for creating project developer

I have read through required fields for project developer formular and added missing ones:
 - picture (active storage)
 - entity_legal_registration_number
 - locations (has many relation)

Then I have added new api endpoint for creating project developer with all required fields -- it creates account, updates user role and creates project developer record. Error is returned in case user already have account.

There are a few tricky parts which I was not sure how to handle and which I will comment at code at this PR. I have also discovered two rswag bugs, which made testing a bit difficult.

## Testing instructions

Rebuild database (drop and create if from scratch) and check swagger api docs. There should be new API endpoint for creating project developers `POST /api/v1/project_developers`

![Screenshot 2022-03-24 at 16 16 00](https://user-images.githubusercontent.com/20660167/159949055-42fc0922-5361-4dec-8e79-13d4b15d2c14.png)

## Tracking

https://vizzuality.atlassian.net/browse/LET-187
